### PR TITLE
WAITP-1516-datatrak-meta-tags

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:

- Update Title, description and image

### CODE REVIEW

File: packages/datatrak-web/index.html [hunk 1]
```diff
@@ -4,9 +4,13 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no maximum-scale=1"  />
     <meta name="theme-color" content="#000000" />
+    <!-- Open Graph for Facebook link preview -->
+    <meta property="og:image" content="https://datatrak.tupaia.org/social-preview.png" />
```
Comment (jaskfla) [inline packages/datatrak-web/index.html:8]: In the spirit of accessibility, should we have an accompanying `og:image:alt`?

Comment (jaskfla) [inline packages/datatrak-web/index.html:8]:
```suggestion
    <meta property="og:image" content="https://datatrak.tupaia.org/social-preview.png" />
    <meta property="og:image:type" content="image/png" />
```

Comment (tcaiger) [reply packages/datatrak-web/index.html:8]: Yes good point. I didn't know about that tag.

Comment (tcaiger) [reply packages/datatrak-web/index.html:8]: Or these ones but they are a good idea so I will add them all in

**Review by acdunham — APPROVED** [review]
Looks great, thank you!

**Pull request merged**